src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const answer = {
+    result: 0,
+    operate: function(callback, number) {
```

Comment:
```suggestion
    operate(callback, number) {
```
not a big mistake, but you can define function in a more simple way